.circleci/config.yml

```diff
@@ -0,0 +1,111 @@
+version: 2.1
+executors:
+  default:
+    working_directory: ~/mattermost-webapp
+    docker:
+      - image: mattermost/mattermost-build-webapp:oct-2-2018
+
+jobs:
+  install:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - dependencies-{{ checksum "package-lock.json" }}
+      - run: |
+          npm ci
+          cd node_modules/mattermost-redux && npm i && npm run build
+      - save_cache:
+          paths:
+            - node_modules
+          key: dependencies-{{ checksum "package-lock.json" }}
+
+  lint:
+    executor:
+      name: default
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - dependencies-{{ checksum "package-lock.json" }}
+      - run: |
+          ls -la node_modules/mattermost-redux
```

Comment:
Is this temporary until the build issue of mattermost-redux is finally solve?  But will not hurt to remain here, just asking 😄

Reply:
yep :) forgot. removing